src/platform/linux/portalgrab.cpp

```diff
@@ -0,0 +1,777 @@
+/**
+ * @file src/platform/linux/portalgrab.cpp
+ * @brief todo
```

Comment:
Every file has this completed now (probably wasn't the case when this PR was first submitted), so plenty of examples.